src/ol/featureloader.js

```diff
@@ -87,8 +87,10 @@ ol.featureloader.tile = function(url, format) {
        * @this {ol.VectorTile}
        */
       function(features, dataProjection) {
-        this.setProjection(dataProjection);
-        this.setFeatures(features);
+        /** @type {ol.VectorTile} */
+        var me = this;
```

Comment:
I try the solution from @gberaudo (and update this pull request),
I also move back the `oli.js`, and `olx.js` back to `--js` in the build and now it's working whell, but I have an other question: the files `oli.js`, and `olx.js` are in the `external`folder but they are not external files in the GCC meaning..., When why they are not named as (with folder) respectively `src/ol/interfaces.js` and `src/ol/export.js`, like the `src/ol/typedef.js`.

And finally we should add this to make everything working:
https://github.com/camptocamp/ngeo/pull/2009/commits/0f2554c3c1e06d0a8a05f0c0c76d447d1c606932#diff-b835803948af99299e646c466f3127f5R6
```
// Required by olx
/** @suppress {extraRequire} */
goog.require('ol.format.IGC');
/** @suppress {extraRequire} */
goog.require('ol.Overlay');
/** @suppress {extraRequire} */
goog.require('ol.control.ScaleLine');
/** @suppress {extraRequire} */
oog.require('ol.source.WMTS');
/** @suppress {extraRequire} */
goog.require('ol.style.Icon');
/** @suppress {extraRequire} */
goog.require('ol.layer.VectorTile');
```
Shouldn't be this also added in the `olx.js` file?